Slider/Assets/Scripts/UI/SelectableSet.cs

```diff
@@ -14,11 +14,24 @@ public class SelectableSet : MonoBehaviour
     [SerializeField] private Selectable[] selectables;
     public Selectable[] Selectables { get => selectables; }
 
+    [SerializeField] private List<SelectableSet> subSelectableSets;
+    public SelectableSet[] SubSelectableSets { get => subSelectableSets.ToArray(); }
+
     private void Awake()
     {
         if (selectables.Length == 0)
         {
             Debug.LogWarning($"A SelectableSet on {gameObject.name} ({gameObject.GetInstanceID()}) contained no selectables. This likely indicates an error.");
         }
     }
+
+    public void AddSubSelectableSet(SelectableSet selectableSet)
```

Comment:
This is for options menu stuff. Since we want to have a specific tab selected at a time but still be in the "Options Menu" (i.e. pressing ESC takes you back to pause) we add the buttons for each sub-menu into the SelectableSet so UINav works